packages/core/src/computers/Map.ts

```diff
@@ -18,9 +18,7 @@ export const Map: Computer = {
     jsFn({
       name: 'mapper',
       value: multiline`
-        item => ({
-          ...item
-        })
+        item => Object.assign({}, item)
```

Comment:
Though this is more performant, lets keep the example as the more straightforward spread syntax and let the user change that to Object.assign if needed